.php_cs

```diff
@@ -1,6 +1,6 @@
 <?php
 
-require_once './vendor/autoload.php';
+require __DIR__.'/vendor/sllh/php-cs-fixer-styleci-bridge/autoload.php';
```

Comment:
You mean this solution? https://github.com/composer/composer/issues/1493#issuecomment-12492276

AFAIK, this is **exactly** what I did: https://github.com/Soullivaneuh/php-cs-fixer-styleci-bridge/blob/v2.1.1/autoload.php#L32-L34

And this solve the issue I got on Puli.